HTML/Default/xmlbrowser.html

```diff
@@ -377,6 +377,10 @@
 								IF !title.defined || title == ''; title = item.title; END;
 								title = title | html | html_line_break;
 								IF item.parseURLs; title = title FILTER parseURIs; END;
+								IF item.wrap;
+									# Put wrapped text into a single element. Improves display of multiline comments & lyrics.
+									title = "<span style = \"display: inline-table;\"> $title </span>";
```

Comment:
Hmm... can't find `inline-table`, but only `inline-block`? (https://developer.mozilla.org/en-US/docs/Web/CSS/display?retiredLocale=de) .

And don't you print the `title` twice in this case?

And I actually don't understand how this helps with multi-line comments.

Reply:
Would `white-space: pre-line;` be what you need here?

Reply:
`inline-table` appears about half way down that page you linked to. Having said that, I will have another go. I can't properly remember why I found `inline-block` lacking, now. And I'll look at `white-space: pre-line`.

`title` is only printed once. The `$title` that you see is being interpolated into a new string value for `title`.

Reply:
The difference I found between `inline-table` and `inline-block` is illustrated in the images below. As you will see, with `inline-table` the top of the display box aligns with the "Lyrics:" label, and it looks right. But with `inline-block` it's the bottom of the display box that aligns with the "Lyrics:" label. THat doesn't look right at all !

Perhaps there's a 'proper' way to do this. I know too little about HTML to know where to look.

![Screenshot 2022-12-16 at 20 24 24](https://user-images.githubusercontent.com/319101/208184069-18bac95a-6f38-424c-bfc7-97931dda94ed.png)
![Screenshot 2022-12-16 at 20 22 58](https://user-images.githubusercontent.com/319101/208184075-08686ce1-84ad-46f0-b540-7de3e0ac3903.png)

Reply:
Oh, I See! I didn't get the layout idea. But this makes sense. Now what I don't like is to have the actual style mixed in with the structure. I'd prefer it if you could eg. define an additional class `.wrappedMenuItem`, then defined

```css
.browsedbListItem .wrappedMenuItem {
   display: inline-table;
}
```

(or whatever is needed to make this work)

That would allow every skin to handle this the way it wants. Thanks!

Reply:
Would you have an example file you test this with?

Reply:
I had a fiddle with `white-space: pre-line;`. In principle it is exactly what is wanted, except that the preceding `html_line_break` filter already "replaces" newline characters with a `<br>`. By "replaces" I think I mean "introduces", because the generated html still seems to contain the newline characters. So `white-space: pre-line;` would end up double counting line breaks. That took a while to notice, and figure out !

I've moved the style from the templates to the three `slimserver.css` style sheets, as suggested. The `EN` is just a placeholder, `Classic` is as expected, and `Default` adds an additional helpful indentation.

I will try and assemble a sensible example file or two and post them over the next 24 hours or so.